kolla-ansible/ansible/roles/triliovault/defaults/main.yml

```diff
@@ -105,6 +105,54 @@ triliovault_ks_services:
       - {'interface': 'internal', 'url': '{{ triliovault_datamover_api_internal_endpoint }}'}
       - {'interface': 'public', 'url': '{{ triliovault_datamover_api_public_endpoint }}'}
 
+triliovault_keystone_user: "dmapi"	
```

Comment:
instead of hardcoded values of triliovault service user ie dmapi and service passwords can we pass those credentials from 
global variable files like in global.yml

eg
triliovault_keystone_user = "dmapi" 
triliovault_keystone_password = "NSqfUIUseG9U23j7ae3ZjiVg43mrnlOll0osgrIc"

database_user = "dmapi" 
database_password = "NSqfUIUseG9U23j7ae3ZjiVg43mrnlOll0osgrIc"

triliovault_rpc_user = "dmapi"
triliovault_rpc_password = "NSqfUIUseG9U23j7ae3ZjiVg43mrnlOll0osgrIc"


triliovault_keystone_user: "{{ triliovault_service_usr }}"
triliovault_keystone_password: "{{ triliovault_keystone_password }}"

database_user: "{{ database_user }}"
database_password: "{{ database_password }}"

triliovault_rpc_user: "{{ triliovault_rpc_user }}"
triliovault_rpc_password: "{{ triliovault_rpc_password }}"